source/Symbol/SwiftASTContext.cpp

```diff
@@ -6152,6 +6152,19 @@ bool SwiftASTContext::IsFixedSize(CompilerType compiler_type) {
   return false;
 }
 
+const llvm::fltSemantics &
+SwiftASTContext::GetFloatTypeSemantics(size_t byte_size) {
+  const size_t bit_size = byte_size * 8;
+  switch (bit_size) {
+    default:
+      return llvm::APFloatBase::Bogus();
```

Comment:
This really should be an assertion failure.  There is no support in the frontend to emit the FP semantics.